lk2nd/device/dts/msm8909/rules.mk

```diff
@@ -1,9 +1,8 @@
 # SPDX-License-Identifier: BSD-3-Clause
 LOCAL_DIR := $(GET_LOCAL_DIR)
 
-ADTBS += \
```

Comment:
Could you reword the subject of the commit to:
```
dts: msm8909: Build all dtb as qcdt
```
along with fixing the panel list up please